config/eslint.config.js

```diff
@@ -0,0 +1,173 @@
+module.exports = {
+  extends: ['airbnb', 'plugin:prettier/recommended'],
+  plugins: ['import', 'jsx-a11y', 'react', 'react-hooks'],
+  parserOptions: {
+    ecmaVersion: 'latest',
+  },
+  env: {
+    browser: true,
+    node: true,
+  },
+  rules: {
+    // eslint
+    'no-unused-expressions': [
+      'error',
+      {
+        allowShortCircuit: true,
+        allowTernary: true,
+        allowTaggedTemplates: true,
+      },
+    ],
+    'no-console': [
+      'error',
+      {
+        allow: ['warn', 'error', 'info'],
+      },
+    ],
+    'no-plusplus': [
+      'error',
+      {
+        allowForLoopAfterthoughts: true,
+      },
+    ],
+    'no-use-before-define': [
+      'error',
+      {
+        functions: false,
+        classes: false,
+        variables: false,
+      },
+    ],
+    'no-restricted-exports': [
+      'error',
+      {
+        restrictedNamedExports: ['then'],
+      },
+    ],
+    'default-case': [
+      'warn',
+      {
+        commentPattern: '^no default$',
+      },
+    ],
+    'arrow-body-style': 'off',
+    'prefer-arrow-callback': 'off',
+
+    // eslint-plugin-import https://github.com/import-js/eslint-plugin-import
+    'import/no-unresolved': 'off',
+    'import/prefer-default-export': 'off',
```

Comment:
Si on décide de preferer les exports nommés :

```suggestion
    'import/prefer-default-export': 'off',
    'import/no-default-export': 'warn',
```

Reply:
Yes j'ai oublié d'activé le `import/no-default-export`